ui/partials/logviewer/lvLogSteps.html

```diff
@@ -5,13 +5,18 @@
        ng-if="showSuccessful === true || step.result !== 'success'"
        class="btn btn-block logviewer-step clearfix {{::getShadingClass(step.result)}}"
        order="{{step.order}}">
-    <span class="pull-left clearfix text-left">
-      {{::step.order+1}}. {{::step.name}}
-    </span>
+    <span ng-if="(step.result === 'unknown')"
+          class="pull-left clearfix text-left unknown-step-warning"
+          title="Log artifact result property is 'unknown'">
+      unknown step result (!)</span>
```

Comment:
With the proposed PR for 'log not found' warnings in PR https://github.com/mozilla/treeherder/pull/867 I might change the `(!)` above to `fa-warning` if it looks reasonable at that size.